compiler+runtime/include/cpp/jank/runtime/obj/inst.hpp

```diff
@@ -0,0 +1,30 @@
+#pragma once
+
+#include <chrono>
+
+#include <jank/runtime/object.hpp>
+
+namespace jank::runtime::obj
+{
+  using inst_ref = oref<struct inst>;
+  using inst_time_point = std::chrono::time_point<std::chrono::system_clock>;
```

Comment:
This is my best guess as to what underlying cpp type should be used to represent an instant.